### PR TITLE
Bump TMD deps and use allowlist directly

### PR DESCRIPTION
**Pull request by eraderna**

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {:paths ["src" "resources"]
  :deps  {org.clojure/clojure        {:mvn/version "1.11.1"}
-         techascent/tech.ml.dataset {:mvn/version "7.007"}
+         techascent/tech.ml.dataset {:mvn/version "7.010"}
```
Comment (otfrom) [inline deps.edn]: there is a 7.011 now. :-)

File: src/witan/gias.clj [hunk 2]
```diff
@@ -537,25 +537,21 @@
...
-     (ds/->dataset in (update-keys (merge {:file-type    :csv
-                                           :separator    ","
-                                           :dataset-name (or file-path resource-file-name)
-                                           :header-row?  true
-                                           :key-fn       key-fn
-                                           :parser-fn    parser-fn}
-                                          options)
-                                   #({:column-allowlist :column-whitelist
-                                      :column-blocklist :column-blacklist} % %))))))
+     (ds/->dataset in (merge {:file-type    :csv
```
Comment (otfrom) [inline src/witan/gias.clj:548]: :+1:

**Review by otfrom — APPROVED** [review]
Probably not a big deal to bump to 7.011 as it will just have overrides in the places it is called. I'm wondering if we need a :dev alias in these libraries to keep us from needing to override all the time.

File: deps.edn (hunk 1 above)
Comment (eraderna) [reply deps.edn]: Have moved TMD & TC to a `:dev` alias.
TMD v7.011 not on maven yet (so hadn't spotted) but is on clojars so bumped.

**Pull request merged**